modules/qna/src/backend/storage.ts

```diff
@@ -139,29 +150,103 @@ export default class Storage {
       }
     }))
 
-    await this.ghost.upsertFile(ROOT_FOLDER, toQnaFile(topicName), serialize(intents))
+    await this.ghost.upsertFile(FLOW_FOLDER, toQnaFile(topicName), serialize(intents))
     return item.id
   }
 
   async getCountPerTopic() {
-    // TODO Make this parallel
-    const qnaFilesPerTopic = await this.ghost.directoryListing(ROOT_FOLDER, 'qna.intents.json')
+    const qnaFilesPerTopic = await this.ghost.directoryListing(FLOW_FOLDER, 'qna.intents.json')
     const qnaPerTopic = {}
     for (const qnaFile of qnaFilesPerTopic) {
-      qnaPerTopic[qnaFile.split('/')[0]] = (await this.ghost.readFileAsObject<Intent[]>(ROOT_FOLDER, qnaFile)).length
+      qnaPerTopic[qnaFile.split('/')[0]] = (await this.ghost.readFileAsObject<Intent[]>(FLOW_FOLDER, qnaFile)).length
     }
     return qnaPerTopic
   }
 
   async deleteSingleItem(topicName: string, itemId: string) {
     await this.ensureIntentsFileExists(topicName)
-    const intents = await this.ghost.readFileAsObject<Intent[]>(ROOT_FOLDER, toQnaFile(topicName))
+    const intents = await this.ghost.readFileAsObject<Intent[]>(FLOW_FOLDER, toQnaFile(topicName))
     const newIntents = intents.filter(x => x.name !== itemId)
-    await this.ghost.upsertFile(ROOT_FOLDER, toQnaFile(topicName), serialize(newIntents))
+    await this.ghost.upsertFile(FLOW_FOLDER, toQnaFile(topicName), serialize(newIntents))
   }
 
-  async count(topicName: string) {
-    const questions = await this.fetchItems(topicName)
-    return questions.length
+  async exportPerTopic(topicName: string) {
+    const tmpDir = tmp.dirSync({ unsafeCleanup: true })
+    const zipName = path.join(tmpDir.name, `${topicName}.tar.gz`)
+    await mkdirp.sync(path.join(tmpDir.name, MEDIA_FOLDER))
+
+    const jsonQnaForBot = await this.ghost.readFileAsObject<Intent[]>(FLOW_FOLDER, toQnaFile(topicName))
+    const jsonQna = jsonQnaForBot.map(i => {
+      const newContent = [...i.metadata.contentAnswers].map(c => removeBotPrefix(c))
+      const newMetadata = { ...i.metadata, contentAnswers: newContent }
+      return { ...i, metadata: newMetadata }
+    })
```

Comment:
```suggestion
    const jsonQna = jsonQnaForBot.map(i => {
      const contentAnswers = [...i.metadata?.contentAnswers ?? []].map(removeBotPrefix)
      const metadata = { ...i.metadata, contentAnswers }
      return { ...i, metadata }
    })
```

Reply:
I was me who changed some Sync functions to async and I forgot to change some, thanks